components/mpas-framework/Makefile

```diff
@@ -296,6 +296,12 @@ intel-mpi:
 	"CPPFLAGS = $(MODEL_FORMULATION) -D_MPI" )
 
 gfortran:
+	GFORTRAN_GTE_10=$$(expr `gfortran -dumpversion | cut -f1 -d.` \>= 10) ;\
+	if [ "$${GFORTRAN_GTE_10}" = "1" ]; then \
+	    FFLAGS="-fallow-argument-mismatch"; \
+	else \
+	    FFLAGS=""; \
+	fi; \
 	( $(MAKE) all \
 	"FC_PARALLEL = mpif90" \
```

Comment:
@philipwjones, do you know of a more elegant way to do this?  If so, I'm keen to learn.

Reply:
No, not really.